database/migrations/2023_09_05_042050_create_profiles_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('profiles', function (Blueprint $table) {
+            //Temperature unit: Celsius
+            $table->id();
+            $table->string('name', 100);
+            $table->float('soak_temperature');
+            $table->unsignedInteger('soak_time'); //Seconds
+            $table->float('reflow_gradient');
+            $table->float('ramp_up_gradient');
+            $table->float('reflow_peak_temp');
+            $table->float('reflow_max_time');
```

Comment:
Since this is measured in seconds it could also be unsigned integer

Reply:
As I didn't know the possible values for this fields I decided to use floats. But I will make the change for unsigned integers.